assets/js/script.js

```diff
@@ -70,20 +70,51 @@ function showPosition(position) {
             "x-rapidapi-key": "7e2b082138msh261301ec7c957a4p1da6fbjsn1aa9a17ef936"
         }
     }).then(function (response) {
-        // console.log(response)
+        console.log(response)
         var restaurantList = response.result.data
 
-        var restaurantListEl = $("<div>").addClass("container");
+        var restaurantListEl = $('<div>').addClass("container-fluid");
         for (let i = 0; i < restaurantList.length; i++) {
             var restaurantId = response.result.data[i].restaurant_id;
-            var restaurantButtons = $("<button>").text(restaurantList[i].restaurant_name).addClass("rest-button row").attr({ "value": restaurantId, "data-zomato": restaurantList[i].restaurant_name });
-            $("body").append(restaurantListEl);
-            restaurantListEl.append(restaurantButtons);
-
+            var restaurantButtons = $("<button type='button' data-toggle='modal'>").text(restaurantList[i].restaurant_name).addClass("rest-button btn col btn-primary").attr({"data-target": "#restaurantModal" + i,"value": restaurantId, "zomato": restaurantList[i].restaurant_name});
```

Comment:
should be "data-zomato" now to account for other browsers. tutor told me non standard html attributes need data- before them in order to work properly across multiple browsers